stl/inc/optional

```diff
@@ -53,10 +53,14 @@ protected:
     _THROW(bad_optional_access{});
 }
 
+struct _Nontrivial_Dummy_type {

```

Comment:
Naming: Should this be a `_Camel_Snake_chimera`? We generally avoid that (with a limited number of justified and less-justified exceptions).

Reply:
Oops - I added the `_Nontrivial` prefix to `_Dummy_type` to make it more specific, and failed to decapitalize `Dummy`.